libs/utils/src/environment_variable.ts

```diff
@@ -26,6 +26,9 @@ export enum BooleanEnvironmentVariableName {
   // For NH pilot: Use dev certs in prod for Java Card auth. Only relevant if ENABLE_JAVA_CARDS is
   // also true.
   USE_DEV_CERTS_IN_PROD = 'REACT_APP_VX_USE_DEV_CERTS_IN_PROD',
+  // Skips election hash checks in various apps to allow using old fixtures in
+  // development even as their respective election definitions change
+  SKIP_ELECTION_HASH_CHECK = 'REACT_APP_VX_SKIP_ELECTION_HASH_CHECK',
```

Comment:
I might suggest a slightly more specific name, e.g. `SKIP_CVR_ELECTION_HASH_CHECK`. When I first saw this variable name, I thought we might be skipping election hash checks for smart cards

Reply:
OK, good call. It is meant to be a more generic purpose flag - it's skipping the election hash checks in scanning interpretation as well, not just for CVRs. Could maybe be `SKIP_IMPORT_ELECTION_HASH_CHECK` but that's also vague.

Reply:
`SKIP_ELECTION_HASH_CHECK_ON_IMPORT` sounds alright to me! I'm also okay with more verbose if need be, e.g.
`SKIP_SCAN_AND_CVR_IMPORT_ELECTION_HASH_CHECKS`

Reply:
Could also make separate env variables for each place we want to skip the hash check if we want to be more targeted.

Reply:
Cool, I'll split the variables.